k8s/pkg/apis/go.mod

```diff
@@ -2,6 +2,31 @@ module github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis
 
 go 1.13
 
-require k8s.io/apimachinery v0.0.0
+require k8s.io/apimachinery v0.17.0
 
-replace k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20191028221656-72ed19daf4bb
+replace (
+	k8s.io/api => k8s.io/api v0.17.0
```

Comment:
Do we still need all these replaces?

Reply:
I tried to remove these but it was not working immediately. If I get it right they are on the way to "fix" the whole versioning story but are not quite there yet.